src/net/mqttHass.cpp

```diff
@@ -169,6 +199,9 @@ void MqttHass::subscribeDevice(const Device* device) {
         this->subscribe(buf);
         sprintf(buf, "%s/cmnd/BRIGHTNESS", device->getName());
         this->subscribe(buf);
+    } else if (device->getMode() == SHUTTER) {
```

Comment:
Good catch, this was missing.
Nevertheless, this could be grouped with the first if as the code is identical.

Reply:
Agree so far, maybe in future I can implement the recall saved position here.